buildSrc/src/main/kotlin/com/datadog/gradle/config/AndroidConfig.kt

```diff
@@ -19,7 +19,7 @@ object AndroidConfig {
     const val MIN_SDK_FOR_WEAR = 23
     const val BUILD_TOOLS_VERSION = "33.0.0"
 
-    val VERSION = Version(1, 19, 0, Version.Type.SrBetaSnapshot(1))
+    val VERSION = Version(1, 20, 0, Version.Type.SrBetaSnapshot(1))
```

Comment:
shouldn't it be `beta 2` for `1.19.0` instead? the main SDK version is `1.19.0`

Reply:
I want to use beta2 for the `public beta` ...so once I have the public beta for dash I will promote it to `beta2`. Not really sure what's the rule here but I find it more useful this way.

Reply:
Hmmm I guess in a way we could switch it to beta2